README.rst

```diff
@@ -99,40 +94,46 @@ Or install from conda-forge channel using conda (in a Anaconda/Miniconda distrib
 
     $ conda install -c conda-forge start_jupyter_cm
 
+Usage
+-----
+
+Create context menu shortcut(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 After installation, enable the context menu entries from a terminal as follows:
 
 .. code:: bash
 
     $ start_jupyter_cm
 
+On Microscoft Windows, the administrator rights are required to add the 
+entry for all users, otherwise the entries will be added only for the 
+current user. In GNOME and OSX only for the current user.
+
+Remove context menu shortcut(s)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 To remove the context menu entries execute the following in a terminal:
 
 .. code::
 
     $ start_jupyter_cm --remove
 
-To uninstall the package:
-
-.. code:: bash
-
-    $ pip uninstall start_jupyter_cm
-
-On Microscoft Windows, the administrator rights are required to add the 
-entry for all users, otherwise the entries will be added only for the 
-current user. In GNOME and OSX only for the current user.
-
 Also, be aware that, uninstalling the package does not
 remove the context menu entries. If you are left with the context menu
 entries after uninstalling ``start_jupyter_cm``, reinstall it, remove
 the entries as above and uninstall it again.
 
-Microsoft Windows
-~~~~~~~~~~~~~~~~~
+Optional parameters
+~~~~~~~~~~~~~~~~~~~
+
+On Linux, several file manager can be installed, to create or remove the context
+menu shortcut(s) for a specific file manager, use the ``--file_manager`` (``-f``) option:
+
+.. code:: bash
+
+    $ start_jupyter_cm -f nautilus
```

Comment:
``nautilus`` -> ``Nautilus``?

Reply:
Or support both?

Reply:
I would go for supporting just the low-case version. Most arguments elsewhere are case-sensitive, so we shouldn't spoil our users ;)

Reply:
Yes, agree done!